src/backend/transaction/views.py

```
@@ -84,3 +84,33 @@ def get_expenses(request):
+    if serializer.is_valid():
+        expense = serializer.save()   #pravimo expense
+    else:
+        return Response(serializer.errors, status=400) #ako ne uspije serializer
+    return Response(ExpenseSerializer(expense).data, status=201)
+
+@api_view(['DELETE'])
+@permission_classes([IsAuthenticated])
+def delete_expense(request, expense_id):
+    profile = request.user.profile
+    try:
+        expense = Expense.objects.get(id=expense_id, profile=profile)
```

Ruff (0.14.11)

`Expense` may be undefined, or defined from star imports

(F405)